omero/developers/build-system.txt

```diff
@@ -175,34 +172,66 @@ products which get built. Namely, the builds for the non-Java components
 stored under :sourcedir:`components/tools`
 are a bit more complex. Each tools component installs its artifacts to
 the tools/target directory which is copied **on top of** the
-OMERO\_HOME/dist top-level distribution directory.
+:file:`dist` top-level distribution directory.
 
-Comments on Ivy
----------------
+Code-signing
+^^^^^^^^^^^^
 
--  Resolvers are key to how Ivy functions. Currently, the default
-   resolver is called "omero-resolver" and simply looks in our
-   repository (``./lib/repository``) for the jars which were downloaded
-   from git. Multi-resolvers can be defined (as granular as for an
-   individual jar) in order to pick up the latest version of whatever
-   library from HTTP, |SSH|, or from the local file system.
+The OMERO source tree includes a self-signed certificate for testing
+OMERO.insight webstart. This is not used by default to avoid problems with
+double signing official releases. The "release-webstart-signed" target can be
+used, alternatively see :source:`docs/hudson/OMERO.sh` and/or
+:ref:`server_webstart_insight_codesign`.
 
--  OMERO\_HOME/lib/cache : in order to determine the transitive closure
-   of all dependencies, Ivy "resolves" each ivy.xml and stores the
-   resolved ivy.xml in its cache (in our build, ``./lib/cache``) to
-   speed up other processes. However, when changing the Ivy
-   configuration (``./etc/ivyconf.xml``) or version number
-   (``etc/omero.properties->omero.version``) the cache can become stale.
-   This should not happen, but currently does. It may be beneficial for
-   the time being to call ``ant clean`` from the top-level build which
-   will delete the cache.
 
+Jenkins
+^^^^^^^
 
-Comments on build.py
---------------------
+The OME project currently uses Jenkins_ as
+a continuous integration server available :jenkins:`here <>`, so many
+binary packages can be downloaded without compiling them yourself. See the :devs_doc:`Continuous Integration documentation <ci-omero.html>` for further details.
 
-``./build.py`` is a complete replacement for your local ant install. In
-many cases on and on most OS, you will be fine running ``ant``. If you
-have any issues (for example ``OutOfMemory``) , please use
-``./build.py`` instead. **However, only use one or the other; do
-not mix calls between the two.**
+Server build
+------------
+
+The default ant target (``build-default``) will build the OMERO system and
+copy the necessary components for a binary distribution to the :file:`dist`
+directory. Below is a comparison of what is taken from the build, where
+it is put, and what role it plays in the distribution.
+
+.. list-table::
```

Comment:
Why is the first 'lib/server' using file mark-up when none of the subsequent items in this column are?